alvr/session/src/settings.rs

```diff
@@ -178,6 +178,8 @@ pub struct VideoDesc {
 
     pub use_10bit_encoder: bool,
 
+    pub sw_thread_count: u32,
```

Comment:
I think this should be advanced

```suggestion
    #[schema(advanced)]
    pub sw_thread_count: u32,
```

Reply:
In that case I was thinking if moving it here would make more sense? What do you think, is it ok as is or should I move it?
![imagem](https://user-images.githubusercontent.com/6501975/154717514-e4e444b8-87a8-4b0c-8c58-3bd3d655787a.png)